ThinDownloadManager/src/main/java/com/thin/downloadmanager/DownloadRequestQueue.java

```diff
@@ -252,6 +252,14 @@ private void initialize(Handler callbackHandler) {
 	}
 
 	/**
+	 * Perform construction with custom thread pool size.
+     */
```

Comment:
Sorry， I use four space instead of tab, so */ is not Align. And I also think initialize method should not be remove, because it is useful for ThinDownloadManager  default constructor.